scripts/run-dev.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# ./scripts/run-dev.sh quickly setup cluster for ACK development. It
+# ensures that a K8s cluster is accessible, then configures the ACK CRD, RBAC
+# but not install the controller
+
+set -Eeo pipefail
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$SCRIPTS_DIR/.."
+
+AWS_SERVICE=$(echo "${AWS_SERVICE:-""}" | tr '[:upper:]' '[:lower:]')
+
+CONTROLLER_NAMESPACE=${CONTROLLER_NAMESPACE:-"ack-system"}
+
+CLUSTER_NAME="ack-dev-cluster-$AWS_SERVICE"
+
+source "$SCRIPTS_DIR/lib/aws.sh"
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/config.sh"
+source "$SCRIPTS_DIR/lib/logging.sh"
+
+source "$SCRIPTS_DIR/controller-setup.sh"
+source "$SCRIPTS_DIR/kind.sh"
+
+ensure_cluster() {
+      info_msg "Creating KIND cluster ..."
+      setup_kind_cluster "$CLUSTER_NAME" "$CONTROLLER_NAMESPACE"
+
+      info_msg "Installing CRDs , common and RBAC manifest..."
+      install_crd_and_rbac "$CONTROLLER_NAMESPACE"
+}
```

Comment:
We already have an `ensure_cluster` method in another file that achieves a similar premise - https://github.com/aws-controllers-k8s/test-infra/blob/main/scripts/run-e2e-tests.sh#L31-L44

This method also uses the config YAML to determine the name, in case the user wants to use a statically named cluster. Rather than redefining this method here, maybe you could modify that one to have a similar effect and then import and use it in this script.

Reply:
refactor here [refactor ensure_cluster](https://github.com/aws-controllers-k8s/test-infra/pull/338/commits/7779b282dfe056b97e0c4549dddf60a6166a1958)